File: backend/src/main/java/com/marcio/springjwt/Entity/Experiencia.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.marcio.springjwt.Entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import lombok.Getter;
import lombok.Setter;

/**
 *
 * @author Usuario
 */
 @Setter
 @Getter
 @Entity
public class Experiencia {
    
   
    @Id
    @GeneratedValue(strategy=GenerationType.SEQUENCE)
    private Long id;
    private String puesto;
    private int cantidadAnios;
    private String descripcion;
    

}

File: FrontEnd/src/app/model/Experiencia.ts

export class Experiencia{
    id?:number;
    puesto: string;
    descripcion: string;
    cantidadAnios: number;
 
    constructor(puesto: string, descripcion: string, cantidadAnios: number){
        
              this.cantidadAnios = cantidadAnios;
              this.descripcion=descripcion;
              this.puesto=puesto;
            
    }
    
 }